src/scratch/hooks/flyout_base.js

```diff
@@ -1,10 +1,34 @@
 /* eslint-disable func-names, no-underscore-dangle */
+
+/**
+ * Corner radius of the flyout background.
```

Comment:
@aaron-binary do we need to add comment for our style? i think the name is obvious.